lms/djangoapps/experiments/views.py

```diff
@@ -16,11 +21,17 @@ class ExperimentDataViewSet(viewsets.ModelViewSet):
     permission_classes = (permissions.IsAuthenticated, IsStaffOrOwner,)
     queryset = ExperimentData.objects.all()
     serializer_class = ExperimentDataSerializer
+    _cached_users = {}
```

Comment:
This is scoped for every request. I am hesitant to optimize any further since I don't anticipate this endpoint being used very frequently.